src/proto/mgmt/mgmt.proto

```diff
@@ -33,7 +33,7 @@ import "acl.proto"; // ACL-related requests
 // Used in gRPC fanout to operate on hosts with multiple ranks.
 message RanksReq {
 	repeated uint32 ranks = 1;
-	float timeout = 2;
+	int64 timeout = 2;
```

Comment:
I still don't understand why these timeouts are being sent as part of the request payload. I see that these were introduced in the PR for DAOS-3647, but it doesn't really make sense to me. It just makes the payload more complex and adds more work for the sender/receiver. Is there some reason why the receiving handlers can't just use the appropriate rank timeouts defined above?